aten/src/ATen/native/sparse/SparseTensorMath.cpp

```diff
@@ -1174,4 +1173,24 @@ Tensor _sparse_sum_backward_cpu(const Tensor& grad_, const SparseTensor& input_,
   }
 }
 
+Tensor isnan_sparse(const Tensor & self){
+  AT_ASSERT(self.is_sparse());
+  SparseTensor out =  at::sparse_coo_tensor({0}, self.options().dtype(at::kBool));
+  out.resize_as_(self);
+  auto indices = out._indices();
+  indices.resize_as_(self._indices());
+  indices.copy_(self._indices());
+  Tensor out_values = out._values();
+  out_values.resize_as_(self._values());
+  Tensor nan_values = at::isnan(self._values());
```

Comment:
It would be much better if `isnan_out` was used here, but it looks like we don't actually have that right now.

Reply:
Yeah